filters/general/filter_55_1Hosts_Pro/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "filterId": "1hosts_pro",
+    "id": 24,
+    "name": "1Hosts (Pro)",
+    "description": "List for blocking pesky ads, trackers, and malware. Strict version: has some minimal breakages, prioritizes privacy & safety (adblocking) over UX e.g blocks graph.facebook.com.",
+    "timeAdded": 1640082678993,
```

Comment:
Please update this number.
exec `new Date().getTime()` in the browser console

Reply:
updated file